right_arm_controller:
  type: "astronaut_controllers/ArucoTrajectoryController"
  end_effector_link: "wrist_right_ft_tool_link"
  base_link: "dummy_base_link"
  joints:
    - arm_right_1_joint
    - arm_right_2_joint
    - arm_right_3_joint
    - arm_right_4_joint
    - arm_right_5_joint
    - arm_right_6_joint
    - arm_right_7_joint

  goal_tolerance: 0.015